src/pretix/presale/templates/pretixpresale/event/fragment_cart.html

```diff
@@ -392,12 +395,20 @@
         {% endfor %}
     {% endfor %}
     {% for fee in cart.fees %}
-        <div class="row cart-row">
-            <div class="col-md-4 col-xs-6">
+        <div role="row" class="row cart-row">
+            <div role="cell" class="col-md-4 col-xs-6">
                 <strong>{{ fee.get_fee_type_display }}</strong>
             </div>
+            {% if download %}
+                <div role="cell"></div>
+            {% else %}
+                <div role="cell"></div>
+                {% if not hide_prices %}
+                    <div role="cell"></div>
+                {% endif %}
+            {% endif %}
```

Comment:
This could be simplified but I deliberately decided to mimick the same if-else-structure as in the columnheader section above.